Fluffed Up/Assets/Scripts/WaveManager.cs

```diff
@@ -153,23 +153,16 @@ public void StartWave()
 
         // Clear the countdown text when finished
         countdownText.text = "";
-        currentWave++; // update wave count
-        UpdateWaveCounter();
-        SpawnEnemy();
-    }
-
-    void UpdateWaveCounter()
-    {
-        waveCounterText.text = "Wave " + currentWave.ToString();
-    }
 
-    void SpawnEnemy()
-    {
-        if (currentWave % 9 == 0)
+        // Increment Wave
+        currentWave++;
+        if (currentWave >= 11)
```

Comment:
## Can we change from:

currentWave++;
if (currentWave >= 11)
{
 currentWave = 1
}

## To:

currentWave = (currentWave + 1) % waveList.Count;
..
..
..
..
EnemyLoader(waveList[currentWave]);

Reply:
Resolved thanks!